link.ld

```diff
@@ -9,12 +9,26 @@ SECTIONS
     *(.text*)
     *(.rodata*) 
 
-    . = ALIGN(PAGE_SIZE);
     _etext = .;
   } > FLASH
 
+  .data : ALIGN(4)
+  {
+    _data = .;
+    *(vtable)
+    *(.data*)
+    _edata = .;
+    _data_len = ABSOLUTE(_edata) - ABSOLUTE(_data);
+    FILL(0xa4a4);
+    /* Calculate padding so that .text + .data is a multiple of 64, as they
+     * will be concatenated into flash for the app image. */
+    . += (64 - (ABSOLUTE(_etext) - ABSOLUTE(_text) + ABSOLUTE(_edata) - ABSOLUTE(_data)) % 64) % 64;
```

Comment:
```suggestion
    . = ALIGN(64);
```
Does this also work? Or difference between physical and virtual addresses cause issues?